Doppler.Currency/Startup.cs

```diff
@@ -64,11 +64,6 @@ public void ConfigureServices(IServiceCollection services)
                     Description = "API for Doppler Currency"
                 });
 
-                var security = new Dictionary<string, IEnumerable<string>>
-                {
-                    { "Bearer", Array.Empty<string>() },
-                };
```

Comment:
Was it not being used?

Reply:
Yees, This variable is not used in anything side